fusionbackend/src/main/java/io/fusion/fusionbackend/service/AssetSeriesService.java

```diff
@@ -71,46 +70,44 @@ public AssetSeries getAssetSeriesByCompany(final Long companyId, final Long asse
                 .orElseThrow(ResourceNotFoundException::new);
     }
 
-    public AssetSeries createAssetSeriesFromAssetTypeTemplate(final Long targetCompanyId,
-                                                              final Long assetTypeTemplateId) {
+    @Transactional
```

Comment:
is it mandatory to set @transactional above every create method with nested objects if it already in the class declaration?